recipes/contrastive-learner/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "contrastive-learner" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/lucidrains/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 720952fba62ba9ccb79ea8fbc5336b851cde1aa865d5bf546478a3bb43c9fad6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytorch
+  run:
+    - python >=3.6
+    - kornia
+    - pytorch
+    - torchvision
+
+test:
+  imports:
+    - contrastive_learner
+
+about:
+  home: "https://github.com/lucidrains/contrastive-learner"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Self-supervised contrastive learning made simple"
+  dev_url: "https://github.com/lucidrains/contrastive-learner"
+
+extra:
+  recipe-maintainers:
+    - psortos
+    - oblute
```

Comment:
You've been removing oblute from the other recipes--do you need to here as well? Otherwise, we'll need confirmation they want to be a maintainer.

Reply:
I will be a maintainer!  He removed me because I was away from my computer last week. We are teammates.